nearcore/src/lib.rs

```diff
@@ -252,15 +251,16 @@ pub fn apply_store_migrations(path: &Path, near_config: &NearConfig) {
         info!(target: "near", "Migrate DB from version 28 to 29");
         migrate_28_to_29(&path);
     }
+    if db_version <= 29 {
+        // version 28 => 29: migrate all structures that use ValidatorStake to versionized version
```

Comment:
bad comment (we migrate 28 -> 29).

and BTW - seems that line 249 above is wrong ..

Reply:
I will introduce separate pr for that